src/content/en/ilt/pwa/introduction-to-service-worker.md

```diff
@@ -127,9 +127,27 @@ navigator.serviceWorker.register('/service-worker.js', {
 });
 ```
 
-In this case we are setting the scope of the service worker to `/app/`, which means the service worker will control requests from pages like `/app/`, `/app/lower/` and `/app/lower/lower`, but not from pages like `/app` or `/`, which are higher.
+In this case we are setting the scope of the service worker to `/app/`, which means the service worker will control requests from pages like `/app/`, `/app/lower/` and `/app/lower/lower`, **but not from pages like `/app` or `/`**, which are higher.
+
+If you want the service worker to control higher pages e.g. `/app` (without the trailing slash) you can indeed change the scope option, but you'll also need to set the [Service-Worker-Allowed HTTP Header](https://w3c.github.io/ServiceWorker/#service-worker-script-response) in your server config for the request serving the service worker script, otherwise you'll get an error. Configuration example:
+
+#### main.js
+
+```
+navigator.serviceWorker.register('/service-worker.js', {
+  scope: '/app'
+});
+```
+
+#### Your server config (e.g. `nginx.conf`)
```

Comment:
Instead of (e.g. `nginx.conf`) in the heading, could you include something along the lines of this in the paragraph, before the example:

> This is an example of adding the `Service-Worker-Allowed` header to the service worker script response using the popular [nginx web server](https://www.nginx.com/resources/wiki/start/topics/examples/full/). If you're using a different web server, please refer to its documentation on setting response headers to make the equivalent change.